src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoLog.kt

```diff
@@ -96,7 +98,7 @@ object AutoLog : Module() {
         mc.connection?.networkManager?.closeChannel(TextComponentString(""))
         mc.loadWorld(null as WorldClient?)
 
-        mc.displayGuiScreen(KamiGuiDisconnected(reasonText, screen, disable.value == DisableMode.ALWAYS || (disable.value == DisableMode.NOT_PLAYER && reason != PLAYER)))
+        mc.displayGuiScreen(KamiGuiDisconnected(reasonText, screen, disable.value == DisableMode.ALWAYS || (disable.value == DisableMode.NOT_PLAYER && reason != PLAYER), LocalTime.now()))
```

Comment:
which format is LocalTime.now(), ISO8601?

Reply:
yes.